packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -548,3 +553,44 @@ const getQueryOptions = (options: { /* @conditional-compile-remove(rooms) */ rol
   }
   return { video: true, audio: true };
 };
+
+const getEndedCallStrings = (
+  locale: CompositeLocale,
+  endedCall?: CallState
+): { title: string; moreDetails?: string; disableStartCallButton: boolean } => {
+  let title = locale.strings.call.leftCallTitle;
+  let moreDetails = locale.strings.call.leftCallMoreDetails;
+  let disableStartCallButton = false;
+  /* @conditional-compile-remove(teams-adhoc-call) */
+  switch (endedCall?.callEndReason?.subCode) {
+    case 10037:
```

Comment:
what do these codes signify and should we store these somewhere rather than hardcoding here?

Reply:
Sorry I didn't see this message before merging. But the codes are detailed here https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/32783/Code-Subcode.